app/graphql/types/job_invocation.rb

```diff
@@ -0,0 +1,16 @@
+module Types
+  class JobInvocation < BaseObject
+    description 'A Job Invocation'
+
+    global_id_field :id
+    field :job_category, String
+    field :description, String
+    field :time_span, Integer
+    field :start_at, GraphQL::Types::ISO8601DateTime
+    field :status_label, String
```

Comment:
What is status label?

Reply:
https://github.com/theforeman/foreman_remote_execution/blob/master/app/models/job_invocation.rb#L134

![status-label](https://user-images.githubusercontent.com/2664090/120769871-d7317c00-c51d-11eb-9105-d89ad8570462.png)

Reply:
Ah, I completely forgot we have this thing. Thank you for reminding me